05/index.html

```diff
@@ -8,6 +8,13 @@
     <title>devmentor.pl - React introduction - #05</title>
 </head>
 <body>
-     
+    <div id="root"></div>
+    <script crossorigin
+src="https://unpkg.com/react@17/umd/react.development.js"
+></script>
+    <script crossorigin
+src="https://unpkg.com/react-dom@17/umd/react-dom.develop
+ment.js"></script>
+<script type="module" src="app.js"></script>
```

Comment:
Podobnie jak wcześniej.

Jeśli uruchomimy webpacka z odpowiednią konfiguracją (zmiana numeru zadania) to wszystko zadziała bez tego.